bootstrap-salt.sh

```diff
@@ -4730,82 +4730,33 @@ install_freebsd_restart_daemons() {
-        [ -z "$MINTIME" ] && MINTIME=$TIME
-
-        FASTER_MIRROR=$(echo "$TIME < $MINTIME" | bc)
+        echodebug "ping time for $MIRROR_HOST is $TIME"
+        if [ -z "$MINTIME" ]; then
+            FASTER_MIRROR=1
+        else
+            FASTER_MIRROR=$(echo "$TIME < $MINTIME" | bc)
+        fi
         if [ "$FASTER_MIRROR" -eq 1 ]; then
             MINTIME=$TIME
-            OPENBSD_REPO=$MIRROR
-        else
-            continue
+            OPENBSD_REPO="$MIRROR"
         fi
     done
 }
 
-
 install_openbsd_deps() {
     __choose_openbsd_mirror || return 1
-    [ -n "$OPENBSD_REPO" ] || return 1
     echoinfo "setting package repository to $OPENBSD_REPO with ping time of $MINTIME"
-    echo "installpath = ${OPENBSD_REPO}${OS_VERSION}/packages/${CPU_ARCH_L}/" >/etc/pkg.conf || return 1
-    pkg_add -I -v lsof || return 1
-    pkg_add -I -v py-pip || return 1
-    __linkfile /usr/local/bin/pip2.* /usr/local/bin/pip $BS_TRUE || return 1
-    __linkfile /usr/local/bin/pydoc2* /usr/local/bin/pydoc $BS_TRUE || return 1
-    __linkfile /usr/local/bin/python2.[0-9] /usr/local/bin/python $BS_TRUE || return 1
-    __linkfile /usr/local/bin/python2.[0-9]*to3 /usr/local/bin/2to3 $BS_TRUE || return 1
-    __linkfile /usr/local/bin/python2.[0-9]*-config /usr/local/bin/python-config $BS_TRUE || return 1
-    pkg_add -I -v swig || return 1
-    pkg_add -I -v py-zmq || return 1
-    pkg_add -I -v py-requests || return 1
-    pkg_add -I -v py-M2Crypto || return 1
-    pkg_add -I -v py-raet || return 1
-    pkg_add -I -v py-libnacl || return 1
-    if [ "$_UPGRADE_SYS" -eq $BS_TRUE ]; then
-        /usr/local/bin/pip install --upgrade pip || return 1
-    fi
-    #
-    # PIP based installs need to copy configuration files "by hand".
-    # Let's trigger config_salt()
-    #
-    if [ "$_TEMP_CONFIG_DIR" = "null" ]; then
-        # Let's set the configuration directory to /tmp
-        _TEMP_CONFIG_DIR="/tmp"
-         CONFIG_SALT_FUNC="config_salt"
-
-         for fname in api master minion syndic; do
-            # Skip salt-api since there is no example config for it in the Salt git repo
-            [ $fname = "api" ] && continue
-
-            # Skip if not meant to be installed
-            [ $fname = "master" ] && [ "$_INSTALL_MASTER" -eq $BS_FALSE ] && continue
-            [ $fname = "minion" ] && [ "$_INSTALL_MINION" -eq $BS_FALSE ] && continue
-            [ $fname = "syndic" ] && [ "$_INSTALL_SYNDIC" -eq $BS_FALSE ] && continue
-
-            # Let's download, since they were not provided, the default configuration files
-            if [ ! -f "$_SALT_ETC_DIR/$fname" ] && [ ! -f "$_TEMP_CONFIG_DIR/$fname" ]; then
-                ftp -o "$_TEMP_CONFIG_DIR/$fname" \
-                    "https://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
-            fi
-        done
-    fi
+    [ -n "$OPENBSD_REPO" ] || return 1
+    echo "installpath = ${OPENBSD_REPO}" >/etc/pkg.conf || return 1
```

Comment:
Here I would append to pkg.conf and installpath.
Something like:
echo "installpath += ${OPENBSD_REPO}" >>/etc/pkg.conf || return 1